### ISSUE TITLE
Add open source andOTP hint in 2FA backend config

### ISSUE THREAD

**Issue by vmario89** [sismics/docs#494]
Hi,

personally i switched over from Google Authenticator to Open Source andOTP https://github.com/andOTP/andOTP which is available on F-Droid and PlayStore. Maybe Teedy could give an additional hint to this and/or other tools handling 2FA secrets. Teedy lists the following atm
![grafik](https://user-images.githubusercontent.com/5631071/103638285-016e1600-4f4d-11eb-94d7-c9f2c56a1363.png)

regards, Mario

**Comment by jendib** [comment]
I agree. All translation files need to be changed and updated to include andOTP. PR is welcome.
